plugins/postcss-font-format-keywords/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@csstools/postcss-font-format-keywords",
+  "version": "1.0.0",
+  "description": "Use unquoted format on @font-face CSS definitions.",
+  "author": "Jonathan Neal <jonathantneal@hotmail.com>",
+  "license": "CC0-1.0",
+  "homepage": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-font-format-keywords#readme",
+  "engines": {
+    "node": "^12 || ^14 || >=16"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
```

Comment:
Can we add the `exports` fields?
I haven't added it yet to the base plugin but I try to add it consistently now.

Reply:
Good catch, will do